src/CollapsibleTabView.tsx

```diff
@@ -322,11 +322,11 @@ const CollapsibleTabView = <T extends Route>({
       const value = event.nativeEvent.layout.height - tabBarHeight;
       if (Math.round(value * 10) / 10 !== Math.round(headerHeight * 10) / 10) {
         onHeaderHeightChange?.();
-        setHeaderHeight(Math.max(value, 0));
```

Comment:
We need to make sure that the value is positive, otherwise, the Animated API will throw an error. That's why we pick the maximum value, between `value` and 0.

Reply:
Doesn't this do that, if not - it defaults back to tabBarHeight?

Reply:
Sorry @PedroBern you were right here. I think this does need to remain like this.